torch/_dynamo/convert_frame.py

```diff
@@ -308,7 +306,7 @@ def _fn(*args: _P.args, **kwargs: _P.kwargs) -> _T:
     return _fn
 
 
-@TorchPatcher.suppress_torch_distributed_warnings
+@TorchPatcher.suppress_torch_distributed_warnings  # type: ignore[misc]
```

Comment:
This error should go away once we land #158276 but we need to add to avoid race conditions for now